src/XMLexport.cpp

```diff
@@ -232,61 +260,61 @@ bool XMLexport::exportHost( QIODevice * device )
     return true;
 }
 
-bool XMLexport::writeHost( Host * pT )
+bool XMLexport::writeHost( Host * pHost )
```

Comment:
`pT` tends to be used for a `TTrigger` or other Mudlet item "type", `pHost` makes things a bit clearer what it points to IMHO.